dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/session/impl/HttpSessionImpl.java

```diff
@@ -56,7 +56,7 @@ public Object getSecurityInfo()
    */
   public Object getFacade()
   {
-    return (Object) _httpSessionFacade;
+    return (Object) returnFacade();
```

Comment:
Why this behavior change?  returnFacade news up a new instance.

Reply:
undo